pyarn/lexer.py

```diff
@@ -35,6 +35,11 @@ def t_NUMBER(t):
     return t
 
 
+# Unquoted string regex (see t_STRING)
+# Docstrings cannot use string interpolation, this value is used by other modules
```

Comment:
Nitpick
```suggestion
# Docstrings can not use string interpolation, this value is used by other modules
```

Reply:
I double-checked some sources, e.g. https://english.stackexchange.com/questions/4510/why-is-cannot-spelled-as-one-word
TLDR both "cannot" and "can not" are acceptable, but "cannot" is more common

Reply:
lol _The modern way of writing can not_ , perhaps I'm old